docs/decisions/0002-deploy-pipelines.md

```diff
@@ -0,0 +1,53 @@
+---
+status: accepted
+date: 2022-11-04
+---
+# Deploy pipeline along with the operator
+
+## Context and Problem Statement
+
+CNBi feature includes OpenShift Pipelines, that are used for task actions.
+The pipelines are not entirely dependent on the CNBi Operator.
+However the CNBi Operator triggers the pipelines execution, so there is a dependency from CNBi Operator end.
+
+## Decision Drivers
+
+* Consistency in deployment
+* Minimize the resource location
```

Comment:
It is a bit unclear (to me at least) what "minimize resource location" means...

Reply:
```suggestion
* Avoid spreading resources in several locations
```
Maybe ?

Reply:
yes, it means we don't want to maintain them separate in different places.